R/FilterStateExpr.R

```diff
@@ -154,7 +156,7 @@ FilterStateExpr <- R6::R6Class( # nolint
     #' `shiny` module server.
     #'
     #' @param id (`character(1)`)
-    #'   `shiny` module instance id
+    #'   `shiny` module instance id.
```

Comment:
in many other methods param description you don't have dot at the end

Please do ctrl+f - this is just my quick spot when scrolling

Reply:
Only in comments, not actual `roxygen`. Comments are also unbroken lines.

Reply:
Fixed.